client/proto/request/v1/request.proto

```diff
@@ -28,12 +29,13 @@ service RequestService {
   // RESOURCE_EXHAUSTED: if Concord Client is overloaded. The caller should retry with a backoff.
   // UNAVAILABLE: if Concord Client is currently unable to process any requests. The caller should retry with a backoff.
   // INTERNAL: if Concord Client cannot progress independent of the request.
+  // ABORTED: this error is returned if the PreExecution read/time conditions are not met. TODO: which preexecution condition was violated?
```

Comment:
@tdimitrov : please expand on these three kinds of error.

Reply:
I think the issue is simply that we are likely to have many more requests open at once. I'm afraid I don't have a good enough understanding of all of the various components to know if it affects anything else.

This may or may not be a problem, but I suggest we pay close attention to any performance impact as a result. Do you think we can test this in isolation to see if it makes any difference to performance under load?